### PR TITLE
docs: application-api.md

### PR DESCRIPTION
**Pull request by watonyweng**
# ⚠️⚠️⚠️ DISCLAIMER ⚠️⚠️⚠️:

Thanks for your interest in submitting a PR! At this time, because the docs are in beta, the team is currently in the midst of changes. As a result, **we are not ready for additional contributions yet**.

## How can I provide feedback?

To be respectful of your time and ideas, **please [create an issue](https://github.com/vuejs/docs-next/issues/new) instead of a pull request**. Otherwise, we will have to simply close your pull request at this time.

Thanks for your understanding!
#159

### CODE REVIEW

File: src/api/application-api.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ import { createApp } from 'vue'
 
 const app = createApp({})
 
-//  注册一个选项对象
+// 注册一个名为my-component的组件
```
Comment (veaba) [inline src/api/application-api.md:38]: 原文是：`register an options object` ，这里是否需要根据上下文语境来描述的必要吗？@Justineo

File: src/api/application-api.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # 应用 API
 
-在 Vue 3 中，全局改变 Vue 行为的 API 现在移动到了由新的 `createApp` 方法所创建的应用实例上。此外，现在它们的影响仅限于该特定应用实例：
+在 Vue 3 中，改变全局 Vue 行为的 API 现在被移动到了由新的 `createApp` 方法所创建的应用实例上。此外，现在它们的影响仅限于该特定应用实例：
```
Comment (Justineo) [inline src/api/application-api.md:3]:
```suggestion
在 Vue 3 中，全局改变 Vue 行为的 API 现在被移到了由新的 `createApp` 方法所创建的应用实例上。此外，现在它们的影响仅限于该特定应用实例：
```

“全局改变”我认为不需要改动，意思应该是“（以）全局（的方式）改动”。

File: src/api/application-api.md [hunk 3]
```diff
@@ -150,12 +150,12 @@ app.directive('focus', {
 
 上面作为 el 参数收到的真实 DOM 元素的蓝图。
 
-#### prevNode
+#### prevNode
```
Comment (Justineo) [inline src/api/application-api.md:153]:
```suggestion
#### prevNode
```

**Review by Justineo — CHANGES_REQUESTED** [review]

File: src/api/application-api.md (hunk 1 above)
Comment (watonyweng) [reply src/api/application-api.md:38]: 那我保留原来的翻译

**Pull request merged**